translate/convert/tbx2po.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright 2006-2007 Zuza Software Foundation
+#
+# This file is part of translate.
+#
+# translate is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# translate is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, see <http://www.gnu.org/licenses/>.
+
+"""Convert TermBase eXchange (.tbx) glossary file into a Gettext PO file
+
+See: http://docs.translatehouse.org/projects/translate-toolkit/en/latest/commands/tbx2po.html
+for examples and usage instructions
+"""
+
+from translate.storage import po, tbx
+
+
+class tbx2po:
```

Comment:
Probably in this case it doesn't make any sense to use a class with just one method. Looks simpler to just have a function.